domains/certificates/Shared/ProjectOriginClients/Registry.cs

```diff
@@ -50,6 +50,37 @@ public static IssuedEvent CreateIssuedEventForProduction(string registryName, Gu
 
         return issuedEvent;
     }
+    public static IssuedEvent CreateIssuedEventForConsumption(string registryName, Guid certificateId, DateInterval period, string gridArea, string assetId, SecretCommitmentInfo commitment, IPublicKey ownerPublicKey)
```

Comment:
This method is almost identical to CreateIssuedEventForProduction. Please apply DRY

Reply:
Corrected